packages/api/src/controllers/IdentitiesController.js

```diff
@@ -82,6 +84,15 @@ class IdentitiesController {
 
     response.send(transfers)
   }
+
+  getWithdrawalsByIdentity = async (request, response) => {
+    const { identifier } = request.params
+    const { page = 1, limit = 10, order = 'asc' } = request.query
+
+    const withdrawals = await this.transactionsDAO.getWithdrawalsByIdentifier(identifier, Number(page ?? 1), Number(limit ?? 10), order ?? 'asc')
```

Comment:
I think it should identitiesDAO, just like in transfers

Reply:
✅